tests/app/helpers.py

```diff
@@ -312,51 +313,44 @@ def services():
-                    {'id': 2, 'slug': 'cloud-software', 'name': 'Cloud software', 'oneServiceLimit': False,
-                     'unitSingular': 'service', 'unitPlural': 'service'},
-                    {'id': 3, 'slug': 'cloud-support', 'name': 'Cloud support', 'oneServiceLimit': False,
-                     'unitSingular': 'service', 'unitPlural': 'service'},
+                    api_stubs.lot(lot_id=1, slug='cloud-hosting', name='Cloud hosting', one_service_limit=False),
+                    api_stubs.lot(lot_id=2, slug='cloud-software', name='Cloud software', one_service_limit=False),
+                    api_stubs.lot(lot_id=3, slug='cloud-support', name='Cloud support', one_service_limit=False),
                 ]
             else:
                 lots = [
-                    {'id': 1, 'slug': 'iaas', 'name': 'Infrastructure as a Service', 'oneServiceLimit': False,
-                     'unitSingular': 'service', 'unitPlural': 'service'},
-                    {'id': 2, 'slug': 'scs', 'name': 'Specialist Cloud Services', 'oneServiceLimit': False,
-                     'unitSingular': 'service', 'unitPlural': 'service'},
-                    {'id': 3, 'slug': 'paas', 'name': 'Platform as a Service', 'oneServiceLimit': False,
-                     'unitSingular': 'service', 'unitPlural': 'service'},
-                    {'id': 4, 'slug': 'saas', 'name': 'Software as a Service', 'oneServiceLimit': False,
-                     'unitSingular': 'service', 'unitPlural': 'service'},
+                    api_stubs.lot(lot_id=1, slug='iaas', name='Infrastructure as a Service', one_service_limit=False),
+                    api_stubs.lot(lot_id=2, slug='scs', name='Specialist Cloud Services', one_service_limit=False),
+                    api_stubs.lot(lot_id=3, slug='paas', name='Platform as a Service', one_service_limit=False),
+                    api_stubs.lot(lot_id=4, slug='saas', name='Software as a Service', one_service_limit=False),
                 ]
-            metaframework = "g-cloud"
-        elif 'digital-outcomes-and-specialists' in slug:
-            lots = [
-                {'id': 1, 'slug': 'digital-specialists', 'name': 'Digital specialists', 'oneServiceLimit': True,
-                 'unitSingular': 'service', 'unitPlural': 'service'},
-            ]
-            metaframework = "digital-outcomes-and-specialists"
 
-        return {
-            'frameworks': {
-                'status': status,
-                'clarificationQuestionsOpen': clarification_questions_open,
-                'name': name,
-                'slug': slug,
-                'lots': lots,
-                'frameworkAgreementVersion': framework_agreement_version,
-                'framework': metaframework,
-            }
-        }
+        elif 'digital-outcomes-and-specialists' in slug:
+            lots = [api_stubs.lot(lot_id=1, slug='digital-specialists', name='Digital specialists',
```

Comment:
I know this is a straight port over to using the stubs, but maybe the other DOS lots should go in here too.